GVRf/Framework/src/org/gearvrf/GVRViewManager.java

```diff
@@ -94,11 +98,20 @@
     GVRActivity mActivity;
     private int mCurrentEye;
 
+	private boolean mScreenshotCenterEnabled = false;
```

Comment:
You could probably do away with the booleans, and simply test for a non-null callback.